base/errorshow.jl

```diff
@@ -92,7 +92,7 @@ function showerror(io::IO, ex, bt; backtrace=true)
 end
 
 function showerror(io::IO, ex::LoadError, bt; backtrace=true)
-    print(io, "LoadError: ")
+    !isa(ex.error, LoadError) && print(io, "LoadError: ")
```

Comment:
```suggestion
```

I'm not even entirely sure that this part of the text is useful at all?

Reply:
True. I'd go with removing `LoadError:` entirely if people agree. Or would that be considered to be a breaking change?